manifests/nodejs.yml

```diff
@@ -566,7 +567,7 @@ tests/:
     test_stats.py:
       Test_Client_Stats: missing_feature
   test_config_consistency.py:
-    Test_Config_ClientIPHeader_Configured: v5.22.0
+    Test_Config_ClientIPHeader_Configured: *ref_4_0_0
```

Comment:
This is wrong, this feature was made available to APM in version `3.13.0` and `2.26.0`

Reply:
Sorry I missed this comment. I addressed this concern in this PR: https://github.com/DataDog/system-tests/pull/3075#discussion_r1768814666